### ISSUE TITLE
X17 NVIDIA issues

### ISSUE THREAD

**Issue by gabbian0** [Cryptominer937/zPoolMiner#24]
I noticed that when the GUI switch on X17 algo, it remains to 0 profit and GPU are not working.
I went into bin/ccminer_alexis78 and tried to set manually the mining and i see i get this error:
GPU #0: result for 00000000 does not validate on CPU! (i have 6 GPU 1080ti and it gives this error to all of them) and something else about illegal memory.
However, my main point is, that i've set the minimum profit to 0 in GUI settings, but the application is ignoring it, keeping to focus on X17 even if it doesn't give any response.
I think, besides the fact that the miner is not working properly, the app should move on another algo when a prolonged 0.00 profit is detected.

![image](https://user-images.githubusercontent.com/36705490/36491119-4f882f96-172a-11e8-890c-056c84460725.png)

**Comment by Cryptominer937** [comment]
This generally happens with a bad overclock or low virtual memory. and if you enable run script on cuda device lost it should reboot the rig if this were to happen.

**Comment by gabbian0** [comment]
i've got no overclock and virtual memory is 11gb x 6 + 4 = 70Gb.
If i activate run script on cuda device lost it would reboot the rig, but when it reboots it would keep trying on x17. My point was that the GUI should detect that miner is not working and skip to next algo.

**Comment by Cryptominer937** [comment]
I agree, it is in my list of modifications to make. Currently my #1 target is getting multipool working with the ability to select pools. I have to finish this before further modifications as right now a lot of things are broken in the development tree.

**Comment by Cryptominer937** [comment]
OK, had another person have this issue and their solution was to increase the fan speed, decrease the core clock.
